__tests__/app.test.js

```diff
@@ -32,3 +32,33 @@ describe("/api/topics", () => {
     });
   });
 });
+
+describe("/api/articles", () => {
+  describe("GET", () => {
+    it("200: it responds with an article object all article properties and author", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toEqual(
+            expect.objectContaining({
+              title: "Living in the shadow of a great man",
```

Comment:
as you're after article with the id of one, i'd also check the object has the id of 1